boards/CodeBridge (CDC)/board_config.h

```diff
@@ -0,0 +1,18 @@
+#ifndef BOARD_CONFIG_H
```

Comment:
Could you use board directory names that are consistent with the current naming scheme and more Unix-friendly? Don't use spaces, capitals, or parentheses.